sno/diff.py

```diff
@@ -209,6 +228,16 @@ def diff_with_writer(
             sys.exit(1)
 
 
+class SpatialReferenceString(StringFromFile):
+    def convert(self, value, param, ctx):
+        value = super().convert(value, param, ctx)
+
+        try:
+            return make_srs(value)
+        except RuntimeError:
+            self.fail(f"Invalid or unknown spatial reference system: {value}")
```

Comment:
any way to get an error message from proj here?

Reply:
ok, added in the RuntimeError's message. The two I've been able to produce are:

```
Error: Invalid value for '--crs': Invalid or unknown coordinate reference system: 'burgers' (OGR Error: Corrupt data)

Error: Invalid value for '--crs': Invalid or unknown coordinate reference system: 'EPSG:999999' (PROJ: proj_create_from_database: crs not found)
```